### DISCUSSION TITLE
When do I need the se_e3 descriptor?

### DISCUSSION THREAD

**Discussion by youngjae142** [deepmodeling/deepmd-kit#1347] [Q&A]
I noticed that there is a new type of descriptor namely se_e3 a three-body descriptor (and I'm very interested).

My question is that,
1. When would it be better to use the new se_e3 descriptor? (or worse when?)
2. Is the se_e3 descriptor better for materials with many types of atomic species? (like ternary compounds?)
(I'm currently training a ternary compound with se_e2_a descriptors, but it seems not as "good" as the binary case.)

I have been waiting for a report about se_e3 from your group, but I just couldn't wait.
Plz help! 😊

**Comment by amcadmus** [comment]
`e2_a` stands for two-body embedding and `e3` stands for three body embedding. 
In the `e2_a` descriptor, the three body information is included in the environment matrix but not in the embedding matrix. 
`e3` introduces three body information in the embedding matrix, thus is more expressive then `e2`. 
In most cases we have tested, the `e2_a` descriptor is powerful enough, but in some cases [like the BCC metal W](https://arxiv.org/abs/2111.04281), one does need `e3` to describe the transition barrier of crew dislocations migration.

**Reply by youngjae142** [reply]
Thanks for your instant answer!
So, would it be the d-orbital's three-body bonding character that requires the three-body embedding matrix?
I know that interpreting the results from such a learning machine is very tricky, but I'm just curious.
